### PR TITLE
fix: Enhance spec "satellite_settings" and parser to support satellite 6.14

### PR DESCRIPTION
**Pull request by huali027**
* Since satellite 6.14, the column "default" has been deleted from the satellite_settings table.

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [x] Is this PR an enhancement?

### CODE REVIEW

**Comment by huali027** [comment]
@xiangce @JoySnow Please help to review, thanks.

File: insights/components/satellite.py [hunk 1]
```diff
@@ -91,3 +91,33 @@ class IsSatellite611(IsSatellite):
...
+    """
+    This component uses ``SatelliteVersion`` combiner
+    to determine the Satellite version. It checks if the Satellite version is 6.14 and later,
+    and raises ``SkipComponent`` when it isn't.
+
+    Raises:
+        SkipComponent: When the Satellite version is earlier than 6.14.
+    """
+    def __init__(self, sat):
+        if sat.major == 6 and sat.minor < 14:
```
Comment (JoySnow) [inline insights/components/satellite.py:107]: Hi @huali027 , the code looks good to me. Jut one concern here. 

I am new to Satellite things, and just from the code, I guess that a "Satellite 5.x" will also hit this `IsSatellite614AndLater` component. 
Do we process only "Satellite 6" systems, or something are already be done to assured for this.

Comment (huali027) [reply insights/components/satellite.py]: @JoySnow I updated it to exclude satellite 5.x as discussed. 
@xiangce Please help to review too.

**Review by JoySnow — APPROVED** [review]

File: insights/components/satellite.py (hunk 1 above)
Comment (JoySnow) [reply insights/components/satellite.py]: Thanks @huali027 , the code looks good to me .

**Pull request merged**